qa/tasks/deepsea.py

```diff
@@ -511,65 +696,103 @@ class CephConf(DeepSea):
         "osd": "osd.conf",
         }
 
+    deepsea_configuration_files = '/srv/salt/ceph/configuration/files'
+
+    targets = {
+        "dashboard": True,
+        "mon_allow_pool_delete": True,
+        "small_cluster": True,
+        "rbd": False,
+        }
+
     def __init__(self, ctx, config):
         deepsea_ctx['logger_obj'] = log.getChild('ceph_conf')
+        self.name = 'deepsea.ceph_conf'
         super(CephConf, self).__init__(ctx, config)
+        self.log.debug("munged config is {}".format(self.config))
 
-    def _ceph_conf_d_full_path(self, section):
+    def __ceph_conf_d_full_path(self, section):
+        ceph_conf_d = self.deepsea_configuration_files + '/ceph.conf.d'
         if section in self.customize.keys():
-            return "{}/{}".format(self.ceph_conf_d, self.customize[section])
+            return "{}/{}".format(ceph_conf_d, self.customize[section])
 
-    def _custom_ceph_conf(self, section, customizations):
+    def __custom_ceph_conf(self, section, customizations):
```

Comment:
I don't get what is this convention is, why double underscore is better?

Reply:
* no underscore: public method (used from outside the class)
* one underscore: helper functions used by the public method
* two underscores: helper functions used by the single-underscore functions